Hestia/app/src/main/java/hestia/UI/DeviceBar.java

```diff
@@ -34,9 +34,14 @@ public void setDevice(Device device) {
 
     public void setLayout(View view, int layoutId, boolean state) {
         hestiaSwitch.addLayout(view, layoutId);
+        hestiaSwitch.getActivatorSwitch().setOnCheckedChangeListener(null);
         hestiaSwitch.getActivatorSwitch().setChecked(state);
-        Log.i(TAG, "Layout changed for: " + device.getName() + " And switch truth is: " +
-                hestiaSwitch.getActivatorSwitch().isChecked());
+        hestiaSwitch.getActivatorSwitch().setOnCheckedChangeListener(hestiaSwitch);
+//        if (hestiaSwitch.getActivatorSwitch().isChecked() != state) {
+//            hestiaSwitch.getActivatorSwitch().setChecked(state);
+//        }
```

Comment:
The comments should be removed, I don't think we should have this in the Release